config/initializers/devise.rb

```diff
@@ -8,7 +8,7 @@
   # confirmation, reset password and unlock tokens in the database.
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
-  # config.secret_key = '05538f9943e3e15d9b261f8077a982d97f4333535a9ec95cb08004e8d69541930726e16d4d7f0bddb7b83580e22ec6d8a4f960316777bf9f8d8cd6af727fa607'
+  config.secret_key = '05538f9943e3e15d9b261f8077a982d97f4333535a9ec95cb08004e8d69541930726e16d4d7f0bddb7b83580e22ec6d8a4f960316777bf9f8d8cd6af727fa607'
```

Comment:
Good eye @tomlouismurphy 

@EvanGrillo could you regenerate the key here and replace that key with an env var?

example:
```
config.secret_key = ENV["SECRET_DEVISE_KEY"]
```

Reply:
I added a .env file to the project locally & changed the file config/initializers/devise.rb